packages/reactDom/test/test_RSC_model.ml

```diff
@@ -434,6 +408,11 @@ let style_as_json () =
   assert_stream stream
     [ "0:[\"$\",\"div\",null,{\"style\":{\"zIndex\":\"34\",\"color\":\"red\",\"background\":\"blue\"}}]\n" ]
 
+let act_with_simple_response () =
+  let response = React.Json (`String "Server Content") in
+  let%lwt stream = ReactServerDOM.act response in
```

Comment:
Just curiosity, where does `act` comes from?

Reply:
Todo: Change to act to the same name react uses on server to can action